### ISSUE TITLE
Sentinel test always returns true

### ISSUE THREAD

**Issue by Jimwoodward** [hashicorp/sentinel-sdk#9]
When running `sentinel test file_name.sentinel` I cannot get my policy to fail when running a set of tests that are designed to fail. All test scenarios are coming back as `PASS`. Below is an example sentinel file, test file, and the verbose output that illustrate what I'm seeing:

**simple.sentinel**
```
main = rule 
{ 
  x == 1
}
```

**test/simple/fail.json**
```
{
    "global": {
        "x": 2
    }
}
```

**sentinel test output**
cmd: `sentinel test -verbose simple.sentinel`

```
PASS - simple.sentinel
  PASS - test/simple/fail.json
    trace:
      FALSE - simple.sentinel:1:1 - Rule "main"
```

**Comment by Jimwoodward** [comment]
I've made this test code more simple and come up with some other tests.

Case 1:
In this case main must be `true` to `pass`. We expect main to be false though and `fail`.
**simple.sentinel**
```
x = 2

main = rule
{
  x == 1
}
```

sentinel test output
cmd: `sentinel test -verbose`
`PASS - simple.sentinel`

I've also tried some other test cases as well, which all work as expected.

Case 2:
In this case main must be `true` to `pass`. We expect main to be `true` and `pass`.
**simple.sentinel**
```
x = 1

main = rule
{
  x == 1
}
```

sentinel test output
cmd: `sentinel test -verbose`
`PASS - simple.sentinel`

Case 3:
In this case main must be `false` to `pass`. We expect main to be `false` and `pass`.
**simple.sentinel**
```
main = rule 
{ 
  x == 2
}
```

**test/simple/test.json**
```
{
    "global": {
        "x": 1
    },

    "test": {
      "main": false
    }
}
```

**sentinel test output**
cmd: `sentinel test -verbose`

```
PASS - simple.sentinel
  PASS - test/simple/test.json
    trace:
      FALSE - simple.sentinel:1:1 - Rule "main"
```

Case 4:
In this case main must be `false` to `pass`. We expect main to be `true` and `fail`.
**simple.sentinel**
```
main = rule 
{ 
  x == 2
}
```

**test/simple/test.json**
```
{
    "global": {
        "x": 2
    },

    "test": {
      "main": false
    }
}
```

**sentinel test output**
cmd: `sentinel test -verbose simple.sentinel`

```
FAIL - simple.sentinel
  FAIL - test/simple/test.json
    expected "main" to be false, got: true

    trace:
      TRUE - simple.sentinel:1:1 - Rule "main"
```

Basically, it's only the first case that doesn't operate as expected

**Comment by Jimwoodward** [comment]
Good news, I submitted a ticket with Hashicorp and they have just gotten back to me. This is a known issue that was reported internally, and it is something that they are working on.

Although the documentation says that the `main` rule is implicitly assumed to be true, this is not the case. The fix for this is to simply explicitly assert that `main` rule be true. In the words of Hashicorp, 

> It seems that the problem is an absent test object is treated as no assertions at all, rather than an implied true assertion for main

If anyone is curious about what this actually needs to look like below is an updated example of my first case:

### Case 1:
In this case main must be `true` to `pass`. We expect main to be `false` though and `fail`.
**simple.sentinel**
```
x = 2

main = rule
{
  x == 1
}
```

**test/simple/test.json**
```
{
    "global": {
        "x": 2
    },

    "test": {
      "main": true
    }
}
```

**sentinel test output**
cmd: `sentinel test -verbose`
```FAIL - simple.sentinel
  FAIL - test/simple/test.json
    expected "main" to be true, got: false

    trace:
      FALSE - simple.sentinel:1:1 - Rule "main"```

**Comment by mitchellh** [comment]
Thanks for the feedback. This is now resolved and will be in the next CLI release coming out shortly! Sorry it took awhile to get to this.
